YoCode/Program.cs

```diff
@@ -73,21 +73,31 @@ private static async Task Main(string[] args)
 
             var projectRunner = await checkManager.PassGatewayChecksAsync(evidenceList);
 
+            evidenceList = await checkManager.PerformChecks(projectRunner);
```

Comment:
If you move this above line 79 the gateway checks are broken. You can't call PerformChecks without checking for null projectRunner.